gost94/tests/mod.rs

```diff
@@ -11,6 +11,71 @@ new_test!(
     fixed_reset_test
 );
 
+#[rustfmt::skip]
+hash_serialization_test!(
+    gost94_crypto_pro_serialization,
+    Gost94CryptoPro,
+    hex!("
+        51aeb30f746350e15ef31472e3914b1b
+        4b9198e0272881ff2401cea8490e5ab2
+        00010000000000000000000000000000
+        00000000000000000000000000000000
+        13131313131313131313131313131313
+        13131313131313131313131313131313
+        01130000000000000000000000000000
+        00000000000000000000000000000000
+        00
```

Comment:
Hm... This redundant byte for eager hashes is certainly an eyesore. It could be worth to add `serialize`/`deserialize` methods to the `block-buffer` crate.